pyfstat/core.py

```diff
@@ -1017,12 +1019,12 @@ def _calculate_predict_fstat_cumulative(
         ----------
         N : int
             Number of timesteps to use between minStartTime and maxStartTime.
+        pfs_input : dict, optional
+            Input kwargs to predict_fstat (alternative to giving label and
+            outdir. Takes preference over those).
```

Comment:
You mean that if this dict contains labels `"label"` or `"outdir"`, they will override those other options? Currently it sounds like the whole dict is just an alternative to those.